core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/HealthType.java

```diff
@@ -81,7 +81,9 @@ public static HealthType get(String key) {
   public static ArrayList<String> getKeys() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)